Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/presencetracing/organizer/create/TraceLocationCreateViewModel.kt

```diff
@@ -111,7 +111,7 @@ class TraceLocationCreateViewModel @AssistedInject constructor(
         )
     }
 
-    private fun String.isTextFormattedCorrectly(max: Int) = trim().length in 1..max && !contains('\n')
+    private fun String.isTextFormattedCorrectly(max: Int) = isNotBlank() && length <= max && !contains('\n')
```

Comment:
`"            This is not blank               "` would pass ?

Reply:
yes, why shouldn't it?

Reply:
it should not be valid , See [specs](https://github.com/corona-warn-app/cwa-app-tech-spec/blob/main/docs/spec/event-registration-client.md#organizer-tracelocation-creation)

Reply:
blanks are not allowed and the text has max length of 255, anything else goes.
see https://github.com/corona-warn-app/cwa-app-tech-spec/pull/127/files#diff-d50804d49c54cd0ea1d31678c893711ac77c8125fd596f5dcf65e429fb0e7b85

Reply:
`and may not have more than 255 characters after removing any leading or trailing whitespace` `isNotBlank` is not trimming is not it ?

Reply:
This applies to Event Creation https://github.com/corona-warn-app/cwa-app-tech-spec/pull/127/files#diff-d50804d49c54cd0ea1d31678c893711ac77c8125fd596f5dcf65e429fb0e7b85R1140

Reply:
The sentence above is less than 255, or what am I missing?

Reply:
I put the trimming in although it is pointless if the UI restricts to 255 characters